generator/test_case_generator.go

```diff
@@ -210,6 +210,11 @@ var testTypeToTestConfig = map[string][]testConfig{
 			terraformDir: "terraform/eks/daemon/windows/2022",
 			targets:      map[string]map[string]struct{}{"arc": {"amd64": {}}},
 		},
+		{
+			testDir:      "./test/metric_value_benchmark",
+			terraformDir: "terraform/eks/daemon/windows/2025",
+			targets:      map[string]map[string]struct{}{"arc": {"amd64": {}}},
+		},
```

Comment:
It might be too early to add these changes since we do not support container insights on Windows 2025.